UI/window-basic-status-bar.hpp

```diff
@@ -3,30 +3,40 @@
 #include <QStatusBar>
 #include <QPointer>
 #include <QTimer>
-#include <util/platform.h>
 #include <obs.h>
+#include <memory>
 
+class Ui_StatusBarWidget;
 class QLabel;
```

Comment:
Is QLabel still used here? I don't see any class members of this type.

Reply:
Fixed